### DISCUSSION TITLE
Questions about Privacy, finances and bridging

### DISCUSSION THREAD

**Discussion by kevonfernando** [stoatchat/discussions#309] [Q&A] [answered]
Hello dear Revolt team, 

I just found your very promising looking Discord alternative and would love to use it. But first I have some questions about this service: 
1. Where are the chats saved?
2. Are messages (text, media, voice, ...) on servers or DMs encrypted? Is it E2EE?
3. Are there classical apps to download, especially on mobile devices?
4. How does this service finance itself? Do you have an equivalent to Discord Nitro or do you rely on donations?
5. Can one bridge a server from Discord to Revolt? Because I will not be able to force everyone over to Revolt immediately.

Answers to this question would help me on deciding if switching to Revolt from Discord is a viable option.

**Comment by Zomatree** [comment, marked as answer]
> Where are the chats saved?

on our servers

> Are messages (text, media, voice, ...) on servers or DMs encrypted? Is it E2EE?

no, in the future we will have the option to make dms E2EE

> Are there classical apps to download, especially on mobile devices?

yes for desktop and android, for ios you have to use the pwa

> How does this service finance itself? Do you have an equivalent to Discord Nitro or do you rely on donations?

currently we are running on donations which we have plenty of

> Can one bridge a server from Discord to Revolt? Because I will not be able to force everyone over to Revolt immediately.

there are bridge bots you can use and in the future we will have a built in bridge to other platforms

**Reply by kevonfernando** [reply]
Thank you for the quick answer. This helps me a lot in understanding Revolt and it’s future.  

> on our servers

Would this not have to be stated in the privacy policy? At least it would be good to know for users. 

> no, in the future we will have the option to make dms E2EE

So that will get better than Discord which is nice. Is E2EE in consideration also for group chats? And are all messages (dms and groups) currently stored at least in an encrypted state or in clear text?

> yes for desktop and android, for ios you have to use the pwa

This is great. Although they could be better advertised on the homepage. The missing native app for iOS users is unfortunate but not too big of an issue for me as I game on PC.

**Reply by Zomatree** [reply]
> Would this not have to be stated in the privacy policy? At least it would be good to know for users.

This is mentioned in our privacy policy https://revolt.chat/privacy 

> So that will get better than Discord which is nice. Is E2EE in consideration also for group chats? And are all messages (dms and groups) currently stored at least in an encrypted state or in clear text?

Group DMs will have the option to be E2EE as well, currently all messages are stored at plain text but are secure inside our databases

**Reply by kevonfernando** [reply]
> This is mentioned in our privacy policy https://revolt.chat/privacy

I am really sorry but I just can not find that  point. If it does not bother you, could you please tell me where exactly it is written. If not it is okay, at least I know now. 

> Group DMs will have the option to be E2EE as well, currently all messages are stored at plain text but are secure inside our databases

I guess that is good enough for now. But E2EE will not be opt-in, right? Because that practice is just nonsense.

**Reply by Zomatree** [reply]
> ### Where is my information stored?
> Since this is an open-source project, we have team members across Europe and North America. Your information is stored on our first-party servers in the United Kingdom.

> 
> If you wish to request deletion of any information stored on our servers, including any requests under GDPR or DMCA, please contact us at [urgent@revolt.chat](mailto:urgent@revolt.chat).

> 
> Exceptions to third-parties service providers which is used to process your data as mentioned above.

**Reply by kevonfernando** [reply]
Oh gosh 😅. I must be blind. I did not realise that this also applies to the chats. Thanks for helping me and sorry for causing you all this trouble.
